satellite/satellitedb/overlaycache.go

```diff
@@ -987,6 +987,87 @@ func (cache *overlaycache) UpdatePieceCounts(ctx context.Context, pieceCounts ma
 	return Error.Wrap(err)
 }
 
+// GetExitingNodes returns nodes in exiting status.
+func (cache *overlaycache) GetExitingNodes(ctx context.Context) (exitingNodes storj.NodeIDList, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	rows, err := cache.db.Query(cache.db.Rebind(`
+		SELECT id FROM nodes
```

Comment:
Should we also check `ExitInitiatedAt` is not `null` ?

Reply:
I think that makes sense!